packages/dds/tree/src/feature-libraries/treeCursorUtils.ts

```diff
@@ -34,7 +44,7 @@ export function singleStackTreeCursor<TNode>(
     root: TNode,
     adapter: CursorAdapter<TNode>,
 ): CursorWithNode<TNode> {
-    return new StackCursor(root, adapter);
+    return new StackCursor(adapter, [], [], [root], 0);
```

Comment:
I guess this API isn't really meant to be used often but would it be worthwhile to have the empty arrays and 0 be default values?

Reply:
there are exactly two call sites to the API (which is not available outside this file), and reordering the arguments so that making those optional would be possible puts them in what (to me anyway) seems like a less logical order.

Anyone who wants those defaults can simply call singleStackTreeCursor.